usfm_tools/support/parseUsfm.py

```diff
@@ -94,7 +94,7 @@ def usfmTokenNumber(key):
 ft = usfmTokenValue("ft", phrase)
 fq = usfmTokenValue("fq", phrase)
 fqa = usfmTokenValue("fqa", phrase)
-fqae = usfmEndToken("fqa")
+fqae = usfmTokenValue("fqae", phrase)
```

Comment:
@richmahn This change needs to be reverted. The master branch is correct, `fqae = usfmEndToken("fqa")`. I just fixed this a couple days ago.